app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PagesController < ApplicationController
   before_action :skip_session_expiration
+  skip_before_action :disable_caching
```

Comment:
would it make sense to have a shared example group for testing the controllers that skip this `before_action`? Or perhaps this is tested indirectly through feature specs?